clang/lib/CodeGen/CodeGenTBAA.h

```diff
@@ -119,6 +120,7 @@ class CodeGenTBAA {
   llvm::Module &Module;
   const CodeGenOptions &CodeGenOpts;
   const LangOptions &Features;
+  MangleContext *MangleCtx;
```

Comment:
This should either be a unique_ptr or you need to delete it in the destructor.

Reply:
Thanks that should be fixed now!